res/xml/preferences_notifications.xml

```diff
@@ -65,4 +51,9 @@
                 android:entries="@array/pref_notification_priority_entries"
                 android:entryValues="@array/pref_notification_priority_values"/>
 
+        <org.thoughtcrime.securesms.components.SwitchPreferenceCompat
+            android:key="pref_key_inthread_notifications"
+            android:title="@string/pref_in_chat_sounds"
```

Comment:
with this setting, you can enable/disable the plipp-plopp send when you are chatting, inside an open chat.  
the option is not directly related to system-notifications, but fits best here (it was also here before)

Reply:
So, is it possible to rename `pref_key_inthread_notifications` to `pref_key_inthread_sounds`? If not, maybe we should put a comment.

Reply:
well, we cannot rename easily as this would break settings with existing users. but i have added a comment and also added a section to make this clearer to the user.

Reply:
i finally made in-chat-sounds respecting global notification on/off as well as mute. an independent setting would rise the question "what is about 'mute'"? cmp https://github.com/deltachat/deltachat-android/pull/1368#issuecomment-633025655

Reply:
:+1: